### PR TITLE
🌐 Add Spanish translation of Advanced User Guide - Events

### PR DESCRIPTION
**Pull request by bezaca**
Hi there :handshake: !
This pull request include the Spanish translation of advanced/events.md

This pull request continues with the Spanish translation of #1218 

I would apreciate it very much if someone can check and review the translation.

### CODE REVIEW

File: docs/es/docs/advanced/events.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
+# Eventos: startup - shutdown
+
+Puedes definir controladores de eventos (funciones) que necesiten ser ejecutados antes de que inicie la aplicación, o cuando la aplicación se este cerrando.
```
Comment (israteneda) [inline docs/es/docs/advanced/events.md:3]:
```suggestion
Puedes definir controladores de eventos (funciones) que necesiten ser ejecutados antes de que inicie la aplicación, o cuando la aplicación se esté cerrando.
```

File: docs/es/docs/advanced/events.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+Estas funciones pueden ser declaradas mediante `async def` o de forma normal `def`.
+
+!!! Warning
+Solamente los controladores de eventos para la aplicación principal serán ejecutados, no se ejecutaran [Sub Aplicaciones - Montaje](./sub-applications.md){.internal-link target=\_blank}.
```
Comment (israteneda) [inline docs/es/docs/advanced/events.md:8]:
```suggestion
Solamente los controladores de eventos para la aplicación principal serán ejecutados, no se ejecutarán [Sub Aplicaciones - Montaje](./sub-applications.md){.internal-link target=\_blank}.
```

File: docs/es/docs/advanced/events.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+## evento <abbr title="inicío">`startup`</abbr>
+
+Para añadir una función que debe ejecutarse antes que la aplicación inicie, debes declararla con el evento <abbr title="inicio">`"startup"`</abbr> :
+
+```Python hl_lines="8"
+{!../../../docs_src/events/tutorial001.py!}
+```
+
+En este caso, el evento <abbr title="inicio">`startup`</abbr> inicializara los elementos de la "base de datos" (como un `diccionario`) con algunos valores.
```
Comment (israteneda) [inline docs/es/docs/advanced/events.md:18]:
```suggestion
En este caso, el evento <abbr title="inicio">`startup`</abbr> inicializará los elementos de la "base de datos" (como un `diccionario`) con algunos valores.
```

File: docs/es/docs/advanced/events.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+Puedes añadir más de una función que controle eventos.
+
+Y tú aplicación no recibirá peticiones hasta que todos los controladores de eventos de tipo <abbr>`startup`</abbr> se hallan completado.
```
Comment (israteneda) [inline docs/es/docs/advanced/events.md:22]:
```suggestion
Y tú aplicación no recibirá peticiones hasta que todos los controladores de eventos de tipo <abbr>`startup`</abbr> se hayan completado.
```

File: docs/es/docs/advanced/events.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+## evento `shutdown`
+
+Para añadir una función que debe ejecutarse cuando la aplicación se esta cerrando, debe declarase con el evento <abbr>`"shutdown"`</abbr>:
```
Comment (israteneda) [inline docs/es/docs/advanced/events.md:26]:
```suggestion
Para añadir una función que debe ejecutarse cuando la aplicación se está cerrando, debe declararse con el evento <abbr>`"shutdown"`</abbr>:
```

File: docs/es/docs/advanced/events.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+
+```Python hl_lines="6"
+{!../../../docs_src/events/tutorial002.py!}
+```
+
+Aquí , el controlador de evento <abbr>`shutdown`</abbr> escribirá una linea de texto <abbr title="Cierre de aplicación">`"Application shutdown"`</abbr> a un archivo `log.txt`.
```
Comment (israteneda) [inline docs/es/docs/advanced/events.md:32]:
```suggestion
Aquí , el controlador de evento <abbr>`shutdown`</abbr> escribirá una línea de texto <abbr title="Cierre de aplicación">`"Application shutdown"`</abbr> a un archivo `log.txt`.
```

File: docs/es/docs/advanced/events.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+
+!!! info
+En la función `open()`, el <abbr title="modo">`mode="a"`</abbr> significa <abbr title="agregar">"append"</abbr>, por lo tanto, la linea sera añadida después de cualquier elemento que halla en el archivo, sin llegar a sobre escribir el contenido previo.
```
Comment (israteneda) [inline docs/es/docs/advanced/events.md:35]:
```suggestion
En la función `open()`, el <abbr title="modo">`mode="a"`</abbr> significa <abbr title="agregar">"append"</abbr>, por lo tanto, la línea será añadida después de cualquier elemento que halla en el archivo, sin llegar a sobreescribir el contenido previo.
```

File: docs/es/docs/advanced/events.md [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+
+!!! tip
+    Observa que en este caso estamos utilizando la función estándar `open()` de Python para interactúa con el archivo.
```
Comment (israteneda) [inline docs/es/docs/advanced/events.md:38]:
```suggestion
    Observa que en este caso estamos utilizando la función estándar `open()` de Python para interactuar con el archivo.
```

File: docs/es/docs/advanced/events.md [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+
+    Entonces , involucramos I/O <abbr title="entrada/salida">(input/output)</abbr>, que requiere "esperar" para que los elementos se escriban en el disco.
```
Comment (israteneda) [inline docs/es/docs/advanced/events.md:40]:
```suggestion
    Entonces, involucramos I/O <abbr title="entrada/salida">(input/output)</abbr>, que requiere "esperar" para que los elementos se escriban en el disco.
```

File: docs/es/docs/advanced/events.md [hunk 10]
```diff
@@ -0,0 +1,47 @@
...
+
+    Pero `open()` no utiliza `async` o `await`.
+
+    Para ello, declaramos un controlador de evento utilizando `def` en vez de `async def`.
```
Comment (israteneda) [inline docs/es/docs/advanced/events.md:44]: Ya que en la línea anterior estamos empleando la palabra `utiliza` en esta línea podríamos utilizar una palabra diferente.
```suggestion
    Para ello, declaramos un controlador de evento usando `def` en vez de `async def`.
```

**Review by israteneda — CHANGES_REQUESTED** [review]
Buen trabajo 📖 dejo algunos comentarios

**Review by israteneda — APPROVED** [review]
✅  Looks good

**Comment by tiangolo** [comment]
This was covered in another PR, so I'll close this one now, thanks for the effort! :cake: :coffee:

**Pull request closed without merging**